packages/broker/test/unit/plugins/operator/InspectRandomNodeService.test.ts

```diff
@@ -0,0 +1,281 @@
+import {
+    FetchRedundancyFactorFn,
+    findNodesForTarget,
+    FindNodesForTargetFn,
+    findTarget,
+    FindTargetFn,
+    InspectRandomNodeService
+} from '../../../../src/plugins/operator/InspectRandomNodeService'
+import { InspectRandomNodeHelper } from '../../../../src/plugins/operator/InspectRandomNodeHelper'
+import { mock, MockProxy } from 'jest-mock-extended'
+import { StreamAssignmentLoadBalancer } from '../../../../src/plugins/operator/StreamAssignmentLoadBalancer'
+import { randomEthereumAddress } from '@streamr/test-utils'
+import { StreamID, StreamPartID, StreamPartIDUtils, toStreamID, toStreamPartID } from '@streamr/protocol'
+import { EthereumAddress, wait, waitForCondition } from '@streamr/utils'
+import { MessageListener, NetworkPeerDescriptor, StreamrClient, Subscription } from 'streamr-client'
+import { createHeartbeatMessage } from '../../../../src/plugins/operator/heartbeatUtils'
+
+const MY_OPERATOR_ADDRESS = randomEthereumAddress()
+const OTHER_OPERATOR_ADDRESS = randomEthereumAddress()
+const SPONSORSHIP_ADDRESS = randomEthereumAddress()
+const STREAM_ID = toStreamID('streamId')
+
+describe(findTarget, () => {
```

Comment:
Add test case where this operator is the only operator

Reply:
renamed test